services/users.service.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable no-useless-catch */
+import jwt from 'jsonwebtoken'
+import bcrypt from 'bcrypt'
+import { v4 } from 'uuid'
+import mailService from '../services/mail.service.js'
+import usersModel from '../models/users.model.js'
+import CustomError from '../errors/customError.js'
+import {
+  CITY_IS_NOT_EXIST,
+  INCORRECT_ACTIVATION_LINK,
+  INVALID_DATA,
+  USER_IS_EXIST
+} from '../errors/types.js'
+import UserDto from '../dtos/user.dto.js'
+import { Roles } from '../db/models/User.js'
+import citiesModel from '../models/cities.model.js'
+import customersModel from '../models/customers.model.js'
+
+const generateAccessToken = (payload) => {
+  return jwt.sign(payload, process.env.JWT_ACCESS_SECRET, { expiresIn: '24h' })
+}
+
+export default {
+  login: async (email, password) => {
+    try {
+      const user = await usersModel.getUserByEmail(email)
+      const userPassword = user.password
+      const validPassword = await bcrypt.compareSync(password, userPassword)
+      if (!user || !validPassword) {
+        throw new CustomError(INVALID_DATA, 400, `Wrong email or password`)
+      }
+      const userDto = new UserDto(user)
```

Comment:
You don't need userDto, just create literal object

const generateTokenPayload = {
id: user.id,
role: user.role,
.....
}